src/main/riddl/Store/Store.riddl

```diff
@@ -17,35 +17,24 @@ context StoreContext is {
   type StoreStates is any of {Draft, Ready, Open, Closed, Deleted}
   type StoreId is Id(StoreContext.Store)
 
-  type RequiredDraftInfo is {
-    name: String?,
-    description: String?,
-    products: SKU*,
-    venue: VenueId?,
-    location: LocationId?,
-    sponsoringOrg: OrganizationId?
-  }
-
-  type OptionalDraftInfo is {
-    event: EventId?
-  }
-
   type Info is {
-    name: String,
-    description: String,
-    products: SKU+,
+    name: String briefly "Required",
```

Comment:
Maybe we need to think about "required". By declaring "name: String" it is by definition required (as opposed to "name: String?" or "name: String*". Perhaps there is a conversation at the RIDDL level? What is the benefit of adding "briefly "Required""?

Reply:
The comment is an instruction for the developer that this field has to be present in a `DraftState`'s `EditableInfo` before moving out of that state, and cannot be removed if out of that state

Reply:
I agree that it is self-evident, but developers were confused at one point by lack of special character for required fields. This is why I created `RequiredDraftInfo` and `RequiredOptionalInfo`, which are being removed here. These annotations help point out to developers that these are Required fields when they are new to RIDDL (and possibly protobuffers & scala as well)